### ISSUE TITLE
Upgrade to 2.0.3 breaks IconButtonProps import

### ISSUE THREAD

**Issue by daviddelusenet** [react-native-elements/react-native-elements#2423]
When upgrading the `react-native-elements` dependency to `2.0.3` the following import doesn't work anymore in `index.d.ts`:

```tsx
import { IconButtonProps } from 'react-native-vector-icons/Icon';
```

This is causing TypeScrip errors in my custom theme since I'm overwriting the color of icons.

Screenshot 2.0.2:
<img width="568" alt="Screenshot 2020-06-29 at 09 10 36" src="https://user-images.githubusercontent.com/1398418/85983791-69fd9880-b9e8-11ea-8820-3e87207f8acd.png">

Screenshot 2.0.3:
<img width="1135" alt="Screenshot 2020-06-29 at 09 10 59" src="https://user-images.githubusercontent.com/1398418/85983823-784bb480-b9e8-11ea-84ac-2ade59f27349.png">

**Comment by siddiboy** [comment]
The recent upgrade is breaking every react-native-vector-icons import. Below is a screenshot of the lock file:
![image](https://user-images.githubusercontent.com/20492995/86050936-38311400-ba72-11ea-92ff-2be5e9d570f5.png)

seems like the dependency of @types/react-native-vector-icons which was there for react-native-elements 2.0.0 but is no longer there.

**Comment by flyingcircle** [comment]
Fixed with 2.0.4
